cmd/commands/subvolume.go

```diff
@@ -40,11 +40,10 @@ var listCmd = &cobra.Command{
 }
 
 var deleteCmd = &cobra.Command{
-	Use:                "delete",
-	Short:              "Deletes a stale subvolume.",
-	DisableFlagParsing: true,
-	Args:               cobra.ExactArgs(3),
-	Example:            "kubectl rook-ceph delete <subvolumes> <filesystem> <subvolumegroup>",
+	Use:     "delete",
+	Short:   "Deletes a stale subvolume.",
+	Args:    cobra.ExactArgs(3),
+	Example: "kubectl rook-ceph delete <filesystem> <subvolume> <subvolumegroup>",
```

Comment:
It will be most intuitive if we specify the arguments from highest level to lowest level. How about subvolume last?
```suggestion
	Example: "kubectl rook-ceph delete <filesystem> <subvolumegroup> <subvolume>",
```

Reply:
this is the usual format that we follow to create and remove subvolume, it would be good in my opinion to keep it as it is.

Reply:
We don't create it in the tool or in rook, I assume just in the csi driver? Sounds good to keep it consistent, can you just show an example of the creation for my curiosity?

Reply:
```bash
sh-4.4$ ceph fs subvolume create
Invalid command: missing required parameter vol_name(<string>)
fs subvolume create <vol_name> <sub_name> [<size:int>] [<group_name>] [<pool_layout>] [<uid:int>] [<gid:int>] [<mode>] [--namespace-isolated] :  Create a CephFS subvolume in a volume, and optionally, with a specific size (in bytes), a specific data pool layout, a specific mode, in a specific subvolume group and in separate RADOS namespace
```

@travisn this is how it looks in ceph CLI